mungegithub/mungers/assign-unassign-handler.go

```diff
@@ -87,9 +88,18 @@ func (h AssignUnassignHandler) Munge(obj *github.MungeObject) {
 
 	toAssign, toUnassign := h.getAssigneesAndUnassignees(obj, comments, fileList, potentialOwners)
 	for _, username := range toAssign.List() {
-		obj.AssignPR(username)
+		err := obj.AssignPR(username)
```

Comment:
Typically we do the glog.Errorf() in the github.go function and be silent here...